backend/apps/ai/management/commands/ai_create_project_chunks.py

```
@@ -0,0 +1,234 @@
+        parser.add_argument(
+            "--project-key", type=str, help="Process only the project with this key"
+        )
+        parser.add_argument("--all", action="store_true", help="Process all the projects")
+        parser.add_argument(
+            "--batch-size",
+            type=int,
+            default=50,
+            help="Number of projects to process in each batch",
+        )
+
+    def handle(self, *args, **options):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)